### PR TITLE
Optimize markdown conversion

### PR DESCRIPTION
**Pull request by finmaster19**
Significantly reduces the memory usage of Markdown parser by calling md.reset() in every iteration when using the same instance of Markdown. This also provides huge speed boost when run against database which has existing rows.

### CODE REVIEW

**Comment by RauliL** [comment]
The biggest optimization in this commit is not having to load every instance of model into memory because
1. Model instances with empty field values are excluded.
2. Only database ID and value of the field is pulled from the database.

File: shuup/core/migrations/0020_md_to_html.py [hunk 1]
```diff
@@ -6,32 +6,30 @@
...
-def md_to_html(apps, schema_editor):
     ProductTranslation = apps.get_model("shuup", "ProductTranslation")
     CategoryTranslation = apps.get_model("shuup", "CategoryTranslation")
     PaymentMethodTranslation = apps.get_model("shuup", "PaymentMethodTranslation")
     ShippingMethodTranslation = apps.get_model("shuup", "ShippingMethodTranslation")
-    update_field(ProductTranslation, "description")
-    update_field(CategoryTranslation, "description")
-    update_field(PaymentMethodTranslation, "description")
-    update_field(ShippingMethodTranslation, "description")
+    update_field(md, ProductTranslation, "description")
```
Comment (chessbr) [inline shuup/core/migrations/0020_md_to_html.py]: this instance could'nt be at the module level?

Comment (RauliL) [inline shuup/core/migrations/0020_md_to_html.py]: What improvements it would provide?

**Comment by tulimaki** [comment]
@finmaster19 @RauliL @suutari-ai Thank you so much!

**Pull request merged**